orttraining/orttraining/python/training/orttrainer_options.py

```diff
@@ -361,12 +373,17 @@ class ORTTrainerOptions(object):
         graph_transformer.gelu_recompute(bool, default False)
         graph_transformer.transformer_layer_recompute(bool, default False)
         graph_transformer.number_recompute_layers(bool, default False)
-        graph_transformer.propagate_cast_ops_level(integet, default -1)
-            Optimize by moving Cast operations if propagate_cast_ops_level is non-negative.
-            Use predetermined list of opcodes considered safe to move before/after cast operation
-            if propagate_cast_ops_level is positive and use propagate_cast_ops_allow otherwise.
-        graph_transformer.propagate_cast_ops_allow(list of str, [])
-            List of opcodes to be considered safe to move before/after cast operation if propagate_cast_ops_level is zero.
+        graph_transformer.propagate_cast_ops_config (dict):
+            graph_transformer.propagate_cast_ops_config.strategy(PropagateCastOpsStrategy, default INSERT_AND_REDUCE)
+                Specify the choice of the cast propagation optimization strategy. The available options are
+                0. Insert-and-reduce cast operations around the nodes with allowed opcodes
```

Comment:
there can't be new lines or paragraphs here. It must be a single line with as many statements separated by '.' as needed, but a single paragraph without bullets. the auto gen doc fail otherwise

Reply:
@thiagocrepaldi, are these changes to orttrainer_options required?

Reply:
@baijumeswani  Not sure if I folllowed, which changes? While we don't delete `ORTTrainer` from our code-base, we should keep them as up to date as possible. My comment above was related to how sphinx generates documentation based on the docstrings. A paragraph for `Arguments: ` section does not accept new lines, bullets and sub-paragraphs